quiz/models.py

```diff
@@ -791,3 +797,46 @@ def add_or_update(tableName,FieldName, FieldValue):
     if result_count == 0:
         tableName.objects.create(**{FieldName: FieldValue})
     return tableName.objects.get(**{FieldName: FieldValue})
+
+GENDER_CHOICES = (
+        ('M', 'Male'),
+        ('F', 'Female'),
+    )
+class RegistrationData(models.Model):
```

Comment:
Instead of using RegistrationData, you can use something meaningful like ProfileDetails.